package.json

```diff
@@ -57,9 +57,9 @@
     "dev": "bun link && concurrently \"bun run esm:watch\" \"bun run cjs:watch\" \"bun run esm:watch:aliases\" \"bun run cjs:watch:aliases\"",
     "build": "bun run clean && bun run build:cjs && bun run build:esm && bun run build:types",
     "clean": "rimraf ./dist/_esm ./dist/_cjs ./dist/_types ./dist/tsconfig",
-    "test": "vitest -c ./tests/vitest.config.ts",
+    "test": "vitest -c ./packages/test/vitest.config.ts",
     "test:watch": "bun run test dev",
-    "playground": "RUN_PLAYGROUND=true vitest -c ./tests/vitest.config.ts -t=playground",
+    "playground": "RUN_PLAYGROUND=true vitest -c ./packages/test/vitest.config.ts -t=playground",
```

Comment:
This was necessary because typescript wanted them in the same directory. It kinda makes sense anyway because in theory this package is standalone and if published it could be used by paymaster or bundler infras, to test. their stuff in a convenient manner. There is some utility to it outside of the sdk if people wanted it. Also nothing inside `src/test` gets included in the sdk builds

Reply:
I could rename 'packages' to 'src' if you think that makes more sense?

Reply:
We would need to write tests for each module and for each SDK feature we have, wouldn't this increase the bundle size if added in "packages" ? What do you mean with "typescript wanted them in the same directory" ?

Reply:
No bundle size is not affected. 'tests' folder is ommitted from the build - all we're discussing is the name of the folder. Can rename it to src if it makes more sense

Reply:
changed to src